backend/src/models/events.py

```diff
@@ -0,0 +1,25 @@
+from typing import TYPE_CHECKING
+from datetime import datetime
+
+from sqlalchemy import DateTime
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+
+from backend.src.models import BaseModel, TimestampAbstractModel
+
+if TYPE_CHECKING:
+    from backend.src.models.performances import Performance
+
+
+class Event(BaseModel, TimestampAbstractModel):
+    name: Mapped[str]
+    date: Mapped[datetime] = mapped_column(
+        DateTime(timezone=True), default=datetime.now, nullable=False
+    )
+    # TODO: should be implemented Enum choice field
+    place: Mapped[str] = mapped_column(default="scena")
+    type: Mapped[list["Performance"]] = relationship(
```

Comment:
It's wrong name I suppose.
type = performance?

When we have rehearsal type?

I think type should be an enum. 
For all types except level and children we will have additional field performance
<img width="515" alt="Screenshot 2023-12-16 at 2 14 54 PM" src="https://github.com/dmitryzhurkovsky/theater-app/assets/62201197/6fdedcdf-ecae-4c18-bc0e-2b44ef77ba1d">